### ISSUE TITLE
Show In Chat Does Not Work For Giphy Attachments

### ISSUE THREAD

**Issue by searchy2** [GetStream/stream-chat-flutter#370]
**Describe the bug**
The "Show In Chat" option does nothing when the attachment is a Giphy.

**What package are you using? What version?**
v1.5.1

**What platform is it about?**
- [X] Android
- [ ] iOS
- [X] Web
- [X] Windows
- [ ] MacOS
- [ ] Linux

**To Reproduce**
1. Attach a Giphy
2. Tap on the image to view
3. Open overflow menu
4. Click Show In Chat

**Expected behavior**
Show In Chat should work for Giphy images or be removed.

**Screenshots**
![Screenshot_20210330-155457](https://user-images.githubusercontent.com/19896174/113214148-a5aae480-923e-11eb-83f7-c0ef227d2e5d.jpg)

**Comment by deven98** [comment]
Hey @searchy2!

The show in chat needs to be implemented through the MessageListView widget. Take a look at https://github.com/GetStream/flutter-samples/blob/main/packages/stream_chat_v1/lib/main.dart where this is implemented. You can try out the sample app here too.

```
MessageListView(
  onShowMessage: (m, c) async {
      final client = StreamChat.of(context).client;
      final message = m;
      final channel = client.channel(
         c.type,
         id: c.id,
      );
      if (channel.state == null) {
        await channel.watch();
     }
     Navigator.pushReplacementNamed(
         context,
         Routes.CHANNEL_PAGE,
         arguments: ChannelPageArgs(
         channel: channel,
         initialMessage: message,
       ),
     );
  },
)
```

**Comment by searchy2** [comment]
This is in a message list view.
Please follow the step by step instructions I provided to reproduce the issue.

**Comment by deven98** [comment]
@searchy2 My bad, I see the issue now. Will give you an update (and likely a fix) on this soon.

**Comment by deven98** [comment]
Hey @searchy2, this bug should be fixed by #373, thanks for bringing this to our attention. I am closing this for now, feel free to reopen if you still face an issue.

**Comment by searchy2** [comment]
Thank you!
